### PR TITLE
Issue/#20

### LINKED ISSUE #20: [Design]add new navbar
新增768<選單及討論選單 更改一些字體

### PR DESCRIPTION
**Pull request by chixxyy**
1.  新增navbar選單與首頁
2. 首頁中的搜尋框
3. 加入主題顏色
4. RWD 768<=

### CODE REVIEW

**Review by ttonylee — APPROVED** [review]

File: templates/layouts/base.html [hunk 1]
```diff
@@ -6,13 +6,57 @@
...
     <div>
         {% block content %}
         {% endblock  %}
     </div>
+
+    <footer class="fixed bottom-0 left-0 w-full p-4 text-center text-white bg-blue-1">
+        <p>
+            <a href="#" class="transition duration-300 ease-in-out hover:text-green-3">&copy; 聯絡我們</a> |
+            <a href="#" class="transition duration-300 ease-in-out hover:text-green-3">&copy; 隱私政策</a> |
+            <a href="#" class="transition duration-300 ease-in-out hover:text-green-3">&copy; 服務條款</a>
```
Comment (imEudora) [inline templates/layouts/base.html]: 版權宣告copyright建議改一下，聯絡我們、隱私政策、服務條款不用加copy
https://zanzan.tw/archives/30463

File: core/urls.py [hunk 2]
```diff
@@ -15,10 +15,11 @@
     2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
 """
 
-from debug_toolbar.toolbar import debug_toolbar_urls
+from django.conf import settings
```
Comment (imEudora) [inline core/urls.py]: 為什麼要import settings?

**Comment by ttonylee** [comment]
poetry.lock file show merge conflict after this branch: issue/#20 removing "django-debug-toolbar" package, 
but the conflict happened on "content-hash" part and it's normal, I leave the newer one hash-value and remove older one to resolve the conflict

File: core/urls.py
Comment (sabina726) [inline core/urls.py:1]: settings 應該不用加

Comment (sabina726) [inline core/urls.py:1]: settings 應該不用加

**Review by sabina726 — CHANGES_REQUESTED** [review]

**Review by imEudora — CHANGES_REQUESTED** [review]

**Review by alextechtrek — APPROVED** [review]
👍🏻

File: core/settings.py [hunk 3]
```diff
@@ -141,6 +136,7 @@
 # Static files (CSS, JavaScript, Images)
 # https://docs.djangoproject.com/en/5.1/howto/static-files/
 
+STATIC_ROOT = os.path.join(BASE_DIR, "staticfiles")
```
Comment (Double-T1) [inline core/settings.py]: 這句應該不需要

File: templates/layouts/base.html (hunk 1 above)
Comment (chixxyy) [reply templates/layouts/base.html]: 我先推上去再解決

**Review by Double-T1 — APPROVED** [review]

**Review by buding033171 — APPROVED** [review]
Ok

**Review by ttonylee — APPROVED** [review]

File: core/urls.py
Comment (ttonylee) [reply core/urls.py:1]: already removed, please kindly review again, thx

File: core/urls.py (hunk 2 above)
Comment (ttonylee) [reply core/urls.py]: already removed, please kindly review again, thx

**Review by imEudora — APPROVED** [review]

**Review by sabina726 — APPROVED** [review]

**Pull request merged**